tracer/src/Datadog.Trace/Ci/Configuration/CIVisibilitySettings.cs

```diff
@@ -206,7 +216,22 @@ internal void SetDefaultManualInstrumentationSettings()
 
         private TracerSettings InitializeTracerSettings()
         {
-            var tracerSettings = new TracerSettings(GlobalConfigurationSource.Instance, new ConfigurationTelemetry());
+            var source = GlobalConfigurationSource.CreateDefaultConfigurationSource();
+            var defaultExcludedUrlSubstrings = string.Empty;
+            if (EnvironmentHelpers.GetEnvironmentVariable(ConfigurationKeys.HttpClientExcludedUrlSubstrings) is { } substrings &&
```

Comment:
Any reason not to use `source` to get this value? That way it can be set through any of the supported configuration methods? 🤔 Otherwise, if they have set this using the JSON (for example), you'll lose the value

Reply:
True, changed in [26d5a34](https://github.com/DataDog/dd-trace-dotnet/pull/5505/commits/26d5a34fe2d52f3ba32e872f691c0f81e246d7c2)